products/postmarketos.md

```diff
@@ -0,0 +1,75 @@
+---
+title: postmarketOS
+category: os
+tags: linux-distribution
+#iconSlug: postmarketOS
+permalink: /postmarketos
+versionCommand: cat /etc/os-release
+releasePolicyLink: https://wiki.postmarketos.org/wiki/Releases
+eolColumn: Security Support
+eolWarnThreshold: 7
+activeSupportWarnThreshold: 7
+releaseColumn: Latest
```

Comment:
```suggestion
releaseColumn: false
```

```suggestion
```

`latest`/`latestReleaseDate` are always equal to `releaseCycle`/`releaseDate`. This means the latest column is not useful and should not be displayed.